keras_hub/src/models/backbone.py

```diff
@@ -186,14 +186,23 @@ def save_to_preset(self, preset_dir):
         saver = get_preset_saver(preset_dir)
         saver.save_backbone(self)
 
+    def get_lora_target_names(self):
+        """Returns list of layer names which are to be LoRA-fied.
+
+        Subclasses can override this method if the names of layers to be
+        LoRa-fied are different.
+        """
+        return ["query_dense", "value_dense", "query", "value"]
+
     def enable_lora(self, rank):
```

Comment:
We could consider exposing this. E.g. `target_names=None`. Would allow some customization.